index.md

```diff
@@ -0,0 +1,80 @@
+# Vanilla Javascript in the Browser Workshop 
+ 
+Welcome to Vanilla Javascript in the Browser workshop hosted by [San Diego JS][san diego js].
```

Comment:
I think the proper spelling is an uppercase S https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=properly+spell+javascript